src/generateFHIRMedicationLines.ts

```
@@ -19,369 +17,508 @@ import type {
-    // Depending of the extension, different load strategies
+    let config: Configuration;
     switch (extension) {
         case "ts":
-        // TODO later find out how to use await() instead ...
-        let module = require(`${path}`).default;
-        config = module() as Configuration;
-        break;
-
-        // It is considered as json by default
+            // TODO later find out how to use await() instead …
+            const module = require(`${path}`).default;
```

Biome (2.4.10)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)